docs/docusaurus/docs/guides/setup/configuring_data_contexts/how_to_convert_an_ephemeral_data_context_to_a_filesystem_data_context.md

```diff
@@ -57,13 +57,7 @@ You can also instantiate an Ephemeral Data Context (for those occasions when you
 
 You can easily check to see if you are working with an Ephemeral Data Context with the following code (in this example, we are assuming your Data Context is stored in the variable `context`):
 
-```python title="Python code"
-from great_expectations.data_context import EphemeralDataContext
-
-# ...
```

Comment:
Question : Is `how_to_explicitly_instantiate_an_ephemeral_data_context` the correct script for `how_to_convert_an_ephemeral_data_context_to_a_filesystem_data_context.md` ? Just checking because `test_script_runner.py` seems to add the same tests for this PR and #7985

Reply:
It is correct, because the convert part is also included in "instantiate" document.